app/src/main/java/net/squanchy/SquanchyApplication.kt

```diff
@@ -29,6 +29,7 @@ class SquanchyApplication : Application() {
         with(applicationComponent.analytics()) {
             enableExceptionLogging()
             initializeStaticUserProperties()
+            trackUserInitiallyNotLoggedIn()
```

Comment:
Are we _always_ tracking not logged in? What happens when we are logged in but at this point we're just not _yet_ aware of it?

Reply:
Oh nvm I checked the implementation and we do some flag setting there. It's not very obvious from the method name tho :)

Reply:
Yeah, I tried to make it explicit with the `initially` part (after long ponderation).
I'm open to suggestions for a better name

Reply:
See on the class itself, I tried coming up with some names